iBudget/static/src/router/routes.js

```diff
@@ -24,49 +32,47 @@ export default [
     {
         path: '/login', component: Login
     },
+    //spending
     {
-        path: '/Spending_registration', component: Spending_registration
-    },
-    {
-        path: '/spend', component: Spend
-    },
-    {
-        path: '/limit', component: Limit
+        path: '/spendings/add', component: Spending_registration
     },
     {
-        path: '/add_spending', component: Icon_getter
+        path: '/spendings/limit_ind', component: Spend
     },
     {
-        path: '/spending_add', component: Spending_add
+        path: '/spendings/limit_group', component: Limit
     },
     {
-        path: '/Financial_goal', component:Financial_goal
+        path: '/spendings/new', component: Spending_add
     },
     {
-        path: '/funds_registration', component: Funds_registration
+        path: '/spendings/history', component: Spending_history
     },
+    //Group
     {
-         path: '/spending_history', component: Spending_history
+        path: '/groups/add', component: Groups_registration
     },
+
+    //funds
     {
-         path: '/track_incomes', component: Income_tracker
+        path: '/funds/new', component: Funds_registration
     },
     {
-        path: '/password_recovery', component: PasswordRecovery
+        path: '/funds/new_goal', component:Financial_goal
     },
-    {
-        path: '/groups_registration', component: Groups_registration
+      {
+        path: '/funds/tracking_goal', component: Goal
     },
+    //incomes
     {
-        path: '/password_recovery/:token', component: PasswordRecovery
-    },
-    {
-        path: '/Income_registration', component: Income_registration
+        path: '/incomes/tracking', component: Income_tracker
     },
+
     {
-        path: '/goal', component: Goal
+        path: '/incomes/add', component: Income_registration
```

Comment:
What is deifference between `/incomes/add` and `/incomes/new` ?

Reply:
In the future we plan to leave one button with name "add". which will allow the user to add an entry to the history and create a new category( this one for income). Now we have separate buttons  one for  create new category (new) an another one for make entry in income history(add).

Reply:
Maybe  better put the plus icon on this button .